### ISSUE TITLE
WCAG (Web Content Accessibility Guidelines) standards.

### ISSUE THREAD

**Issue by danlatorre** [planetary-social/planetary-ios#1067]

**Comment by danlatorre** [comment]
I posted screen recordings of color contrast changes to make, just need the design choices for the values then we should be good for thin lean way to get to AA and AAA WCAG compliance.
